### ISSUE TITLE
Why use only one of netty's work threads

### ISSUE THREAD

**Issue by yu199195** [reactor/reactor-netty#867]
Hi，I am using webflux, and his bottom layer USES reactor-netty. I have customized the configuration of netty，Why only the same work thread handles each request？

This is my code ：

```
private static class EventLoopNettyCustomizer implements NettyServerCustomizer {

        @Override
        public HttpServer apply(HttpServer httpServer) {
            return httpServer
                    .tcpConfiguration(tcpServer -> tcpServer.bootstrap(
                            server -> {
                              
                                    boosGroup = new NioEventLoopGroup(1);
                                    workGroup = new NioEventLoopGroup(32);
                                    server.channel(NioServerSocketChannel.class);
             
                                 return server.group(boosGroup, workGroup)
                                     
                                        .option(ChannelOption.SO_BACKLOG, 65535)
                                      
                                        .option(ChannelOption.SO_REUSEADDR, true)
                                    
                                        .option(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT)
                                      
                                        .childOption(ChannelOption.TCP_NODELAY, true)
                                        .childOption(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT);

                            }
                    ));
        }
    }
```

**Comment by violetagg** [comment]
@yu199195 Can you provide more complete example?
With that snippet I'm not able to reproduce the issue.

btw you can use the API provided by Reactor Netty instead of configuring the bootstrap directly.

```
.tcpConfiguration(tcpServer -> tcpServer
	.runOn(LoopResources.create("test", 1, 32, true), false)
	.selectorOption(ChannelOption.SO_BACKLOG, 65535)
	.selectorOption(ChannelOption.SO_REUSEADDR, true)
	.selectorOption(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT)
	.option(ChannelOption.TCP_NODELAY, true)
	.option(ChannelOption.ALLOCATOR, PooledByteBufAllocator.DEFAULT))
```

**Comment by yu199195** [comment]
if  set selectorOption(ChannelOption.SO_BACKLOG, 65535)

io.netty.bootstrap.ServerBootstrap - Unknown channel option 'SO_BACKLOG' for channel '[id: 0xbdd7ce90, L:/0:0:0:0:0:0:0:1:8088 - R:/0:0:0:0:0:0:0:1:49303]'

**Comment by yu199195** [comment]
I mean, I set up 32 work threads and found that each time I processed the same business work threads
In my knowledge, netty's work thread runs several to handle requests.
So, I have a little doubt, would like to ask you, why my situation.

**Comment by violetagg** [comment]
@yu199195 any example?

**Comment by violetagg** [comment]
@yu199195 Any update?

About the exception that you see - we had such issue but this was fixed in 0.8.7/0.9.0
#706

**Comment by violetagg** [comment]
Closing this as there is no response from the author
